source/get-started.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+title: Get started
+weight: 20
+last_reviewed_on: 2022-09-01
+review_in: 3 months
+---
+
+# Get started
+
+You will need a [GitHub account with Ministry of Justice organisation access](https://technical-guidance.service.justice.gov.uk/documentation/standards/storing-source-code.html#github-access).
+
+Afterwards, please request to join the [@ministryofjustice/hmpps-interventions-dev](https://github.com/orgs/ministryofjustice/teams/hmpps-interventions-dev) GitHub team.
+
+## GitHub repositories
+
+The service has small components in separate GitHub repositories.
+
+Live components:
```

Comment:
This page is new too

Reply:
No contest, although the naming of the UI component is unfamiliar to me

Reply:
Fair enough. What should we call it?

Reply:
Fixed in 86d320adb02d6c284967e61d4e30db9213fec1ea -- renamed to "Refer and monitor ... UI"